### PR TITLE
Develop dennis

### PR DESCRIPTION
**Pull request by adobrzhansky**

### CODE REVIEW

**Comment by adobrzhansky** [comment]
👍 👍

File: my_test_app.py [hunk 1]
```diff
@@ -3,12 +3,51 @@
...
+    params = {}
+    api_key = None
+    city = None
+    city_map = {'odessa': '46.4736019,30.603976'}
+
+    def __init__(self, api_key, city='odessa', params='currently,minutely,hourly'):
+        self.__check_city(city)
+        self.set_api_key(api_key)
+        self.city = city
+        self.__check_params(params)
```
Comment (ktotok) [inline my_test_app.py:18]: check method should whether contain assertion or returns bool, and not change any attributes implicitly

File: my_test_app.py [hunk 2]
```diff
@@ -3,12 +3,51 @@
...
+    def __check_params(self, params):
+        if len(str(params)) > 0:
+            self.params = {'exclude': params}
+        else:
+            self.params = {}
+
+    def current(self, city_name):
+        self.__check_city(city_name)
+        api_key = self.api_key
+        params = {'exclude': 'currently,minutely,hourly'}
```
Comment (ktotok) [inline my_test_app.py:40]: current params set returns daily forecast

File: my_test_app.py [hunk 3]
```diff
@@ -3,12 +3,51 @@
...
+    def execute_request(self, api_key, coordinates, params):
+        url = self.url.format(api_key=api_key, coordinates=coordinates)
+        response = requests.get(url, params=params)
+        return response
+
+    def __check_city(self, city):
+        assert self.city_map.get(city) is not None, 'No such city in list!'
+
+    def __check_params(self, params):
+        if len(str(params)) > 0:
```
Comment (adobrzhansky) [inline my_test_app.py:32]: how about to `if params:` check?

File: my_test_app.py [hunk 4]
```diff
@@ -3,12 +3,51 @@
...
+    def set_api_key(self, api_key):
+        self.api_key = api_key
+
+    def execute_request(self, api_key, coordinates, params):
+        url = self.url.format(api_key=api_key, coordinates=coordinates)
+        response = requests.get(url, params=params)
+        return response
+
+    def __check_city(self, city):
+        assert self.city_map.get(city) is not None, 'No such city in list!'
```
Comment (adobrzhansky) [inline my_test_app.py:29]: `self.city_map.get(city) is not None` could be replaced for `if self.city_map.get(city)`

**Pull request closed without merging**